### PR TITLE
Front end web updt

### PR DESCRIPTION
**Pull request by andyfire9**
This should work for now, as it stands i will severely work it out in the following days , for now it has a place holder as I wasn't able to get it to represent the comment system but i tested it in the background and it surprisingly works.  The User login system as well as the home should be up for back end functionality modificaiton.

### CODE REVIEW

**Review by kajahno — COMMENTED** [review]
There is no way I can validate the code here, it's a massive PR. 

The style is mixed up, sometimes we use class-based components and sometimes function-based components.

Comments is something that it's hard to test without having the posts, so I would first implement the posts, then work in the comments

File: .gitignore [hunk 1]
```diff
@@ -64,3 +64,4 @@ node_modules/
 
 # dotenv environment variables file
 .env
+front-end/src/pages/Home.jsx
```
Comment (kajahno) [inline .gitignore]: Esto nos dio problemas la ultima vez. Recuerden que los archivos en el .gitignore son archivos que van a ser ignorados por git, por lo cual no van a ser subidos al repositorio. Por ende, tienen que borrar esta linea

File: front-end/src/pages/login.jsx [hunk 2]
```diff
@@ -51,7 +52,7 @@ class login extends Component {
     constructor(props) {
         super(props);
         this.state = {
-            email: "",
+            Username: "",
```
Comment (kajahno) [inline front-end/src/pages/login.jsx]: why uppercase when everything else is lowercase?

File: front-end/src/pages/login.jsx [hunk 3]
```diff
@@ -64,13 +65,25 @@ class login extends Component {
...
         const userData = {
-            email,
+            Username,
             password,
         };
         // TODO: implement creating the user here using axios
+
+        axios.post("/user",userData,{
+            Username: "DennBryant",
+            password: "Ros@23",
```
Comment (kajahno) [inline front-end/src/pages/login.jsx]: La idea es usar las variables del formulario aqui, esto significa que solamente un usuario puede hacer login

File: front-end/src/pages/signup.jsx [hunk 4]
```diff
@@ -44,12 +44,13 @@ const styles = {
...
     },
 };
-
+     // I just add a Birthday TextField in sign up - Denny 
 class signup extends Component {
     constructor(props) {
         super(props);
         this.state = {
             email: "",
+            Birthday:"",
```
Comment (kajahno) [inline front-end/src/pages/signup.jsx]: again, why uppercase when everything else is lowercase?

File: front-end/src/pages/signup.jsx [hunk 5]
```diff
@@ -61,13 +62,15 @@ class signup extends Component {
...
-        const { email, password, confirmPassword, username } = this.state;
+        const { email,Birthday, password, confirmPassword, username} = this.state;
 
         const newUserData = {
             email,
+            Birthday,
             password,
             confirmPassword,
-            username,
+            username
```
Comment (kajahno) [inline front-end/src/pages/signup.jsx]: now username is inconsistent here (it has uppercase somewhere else)

**Comment by andyfire9** [comment]
Basically the function is now in system, the comment system IS the post system , i mentioned this several times before. It takes user id's and Username's to track whoever uploaded a comment and the like, it can be as many posts at a time as we need by simply adding more repetitions, it all gets caled into the home via a post system. While I connected it to the server, the Actual username's and those systems need to be linked to the api contract which it did not allow me too. Other than that it is a nearly end to end functioning system.

**Comment by DennBryant** [comment]
As we have a variable that is called, then the user information can be obtained. For example, when the user is entered in the login, it will be saved in the post request established, for that I put the object and since I have an object from where I can obtain the information What do I need, the dot operator is in charge of getting the information and not only one user can login.

**Comment by kajahno** [comment]
Again, I am not convinced this works. Demo it to me and let's take it from there.

**Pull request closed without merging**